Pin Dropper/PinsListViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PinsListViewController.swift
+//  Pin Dropper
+//
+//  Created by Talha Babar on 7/2/15.
+//  Copyright (c) 2015 Talha Babar. All rights reserved.
+//
+
+
+class PinsListViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var passed : [Pin]!
```

Comment:
`passed` is not a correct name for the `Pin`s passed from the previous VC. `droppedPins` would be a suitable name